--- a/lib/settings.vala
+++ b/lib/settings.vala
@@ -33,7 +33,13 @@ namespace Pomodoro
     public unowned GLib.Settings get_settings ()
     {
         if (Pomodoro.settings == null) {
-            Pomodoro.settings = new GLib.Settings ("org.gnome.pomodoro");
+            try {
+                SettingsSchemaSource schemaSource = new SettingsSchemaSource.from_directory ("@gschemasCompiled@", SettingsSchemaSource.get_default (), true);
+                SettingsSchema schema = schemaSource.lookup ("org.gnome.pomodoro", false);
+                Pomodoro.settings = new Settings.full (schema, null, null);
+            } catch (Error ex) {
+                // Don't care
+            }
         }
 
         return Pomodoro.settings;
